Havtorn/Source/Engine/ECS/Components/CameraControllerComponent.h

```diff
@@ -19,6 +19,8 @@ namespace Havtorn
 			: SComponent(std::move(entity), type)
 		{}
 
+		static constexpr F32 MaxPitchDegreesPerFrame = 90.0f;
```

Comment:
It's not "per frame" though, it's a general max?

Reply:
It's not? My bad.